components/bulletins/BulletinCard.vue

```diff
@@ -55,7 +55,8 @@ export default {
 
 <style lang="postcss" scoped>
 .bulletinCard {
-    @apply relative flex flex-col items-center font-serif rounded-3xl border-4;
+    @apply relative flex flex-col items-center font-serif rounded-3xl;
+    border-width: 3px;
```

Comment:
Sorry for the misinterpretation.
In smaller screen the border with is 3px and in bigger screen the border width enlarges to 4px.